packages/react/src/themes/teams/components/Toolbar/toolbarStyles.ts

```diff
@@ -1,18 +1,24 @@
 import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '../../../types'
+import { ToolbarVariables } from './toolbarVariables'
+import { ToolbarProps } from '../../../../components/Toolbar/Toolbar'
 
-const toolbarStyles: ComponentSlotStylesPrepared = {
-  root: (): ICSSInJSStyle => ({
+const toolbarStyles: ComponentSlotStylesPrepared<ToolbarProps, ToolbarVariables> = {
+  root: ({ props: { overflow }, variables: v }): ICSSInJSStyle => ({
```

Comment:
```suggestion
  root: ({ variables: v }): ICSSInJSStyle => ({
```

nit, I don't see that `overflow` is used anywhere